pkg/auth/rbac.go

```diff
@@ -63,9 +66,28 @@ type Identity struct {
 type rbacConnection struct {
 	srcIdentity Identity
 	dstNetwork  string
-	srcIp       []byte
-	dstIp       []byte
-	dstPort     uint32
+	// srcIp is big endian
+	srcIp []byte
+	// dstIp ip is big endian
+	dstIp []byte
+	// dstPort is little endian
+	dstPort uint32
+}
+
+type bpfSockTupleV4 struct {
+	// All fields are big endian
+	SrcAddr uint32
+	DstAddr uint32
+	SrcPort uint16
+	DstPort uint16
+}
+
+type bpfSockTupleV6 struct {
+	// All fields are big endian
+	SrcAddr [4]uint32
+	DstAddr [4]uint32
```

Comment:
should we use netip.Address @nlgwcy is implementing the ipv6 support with this type